lib/bitstyles_phoenix/bitstyles.ex

```diff
@@ -14,12 +14,51 @@ defmodule BitstylesPhoenix.Bitstyles do
   """
   def classname(name), do: classname(name, version())
 
-  def classname(class, version) when version > "4.3.0" do
+  def classname(class, version) when version > "5.0.1" do
     IO.warn("Version #{version} of bitstyles is not yet supported")
     class
   end
 
-  def classname(class, version) when version >= "4.2.0", do: class
+  def classname(class, version) when version >= "5.0.0" do
+    class
+  end
+
+  def classname(class, version) when version >= "4.2.0" do
+    sizes_renaming = %{
+      "3xs" => "xxxs",
+      "2xs" => "xxs",
+      "2xl" => "xxl",
+      "3xl" => "xxxl"
+    }
+
+    class =
+      Enum.reduce(sizes_renaming, class, fn {new_size, old_size}, acc ->
+        String.replace(acc, "-#{new_size}", "-#{old_size}")
```

Comment:
Shouldn't we at least make sure here, that the class starts with `u-` and is a bitstyles class?

Reply:
[`15d7c55` (#125)](https://github.com/bitcrowd/bitstyles_phoenix/pull/125/commits/15d7c55183b2924d2aa500b113b8aaf72092c99c)